src/include/com/amazonaws/kinesis/video/common/Include.h

```diff
@@ -137,6 +139,8 @@ extern "C" {
  */
 #define MAX_IOT_THING_NAME_LEN MAX_STREAM_NAME_LEN
 
+#define MAX_ECS_TOKEN_LEN 2048

```

Comment:
Add a comment with a link to the AWS limit docs

Reply:
Actually, I did not find any official document on this, so I set this temporarily. Would you mind to provide the link to me?

Reply:
Perhaps someone from KVS could help with that